diff -Nur themes/Ganymede.orig/borders/DEFAULT/border.cfg themes/Ganymede/borders/DEFAULT/border.cfg
--- themes/Ganymede.orig/borders/DEFAULT/border.cfg	1999-10-08 22:43:37.000000000 +0200
+++ themes/Ganymede/borders/DEFAULT/border.cfg	2000-12-28 19:08:45.000000000 +0100
@@ -8,19 +8,20 @@
 
 */
 
+
 __TCLASS __BGN
   __NAME "GANYMEDE"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 200 200 200
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 450
-  __NORMAL_ACTIVE "ganymede/10"
+  __NORMAL_ACTIVE "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 450
-  __NORMAL_ACTIVE_STICKY "ganymede/10"
+  __NORMAL_ACTIVE_STICKY "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/GREEN/border.cfg themes/Ganymede/borders/GREEN/border.cfg
--- themes/Ganymede.orig/borders/GREEN/border.cfg	1999-10-17 03:40:21.000000000 +0200
+++ themes/Ganymede/borders/GREEN/border.cfg	2000-12-28 19:08:57.000000000 +0100
@@ -10,17 +10,17 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_GREEN"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 200 255 200
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 450
-  __NORMAL_ACTIVE "ganymede/10"
+  __NORMAL_ACTIVE "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 200 255 255
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 450
-  __NORMAL_ACTIVE_STICKY "ganymede/10"
+  __NORMAL_ACTIVE_STICKY "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 0 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/MENU_BORDER/border.cfg themes/Ganymede/borders/MENU_BORDER/border.cfg
--- themes/Ganymede.orig/borders/MENU_BORDER/border.cfg	1999-10-27 07:48:51.000000000 +0200
+++ themes/Ganymede/borders/MENU_BORDER/border.cfg	2000-12-28 19:09:14.000000000 +0100
@@ -10,12 +10,12 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_SMALL_ITALIC"
-  __NORMAL "ganymede_italic/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 512
-  __NORMAL_ACTIVE "ganymede_italic/10"
+  __NORMAL_ACTIVE "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/PAGER/border.cfg themes/Ganymede/borders/PAGER/border.cfg
--- themes/Ganymede.orig/borders/PAGER/border.cfg	1999-09-27 19:39:52.000000000 +0200
+++ themes/Ganymede/borders/PAGER/border.cfg	2000-12-28 19:09:26.000000000 +0100
@@ -10,7 +10,7 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_CENTERED_SMALL"
-  __NORMAL "ganymede/7"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/PAGER_LEFT/border.cfg themes/Ganymede/borders/PAGER_LEFT/border.cfg
--- themes/Ganymede.orig/borders/PAGER_LEFT/border.cfg	1999-11-21 20:31:54.000000000 +0100
+++ themes/Ganymede/borders/PAGER_LEFT/border.cfg	2000-12-28 19:09:35.000000000 +0100
@@ -1,16 +1,17 @@
 /*
+Jan 27 2000, Japanese Localization by Masahiko Mori <masa@engr.sgi.com>
+                      Traslation by Ryuichi Wakita <wakita@xtrans.com>
 
-   T  H  E     G  A  N  Y  M  E  D  E      T  H  E  M  E
-
-	  (c) Chris Kreibich, kreibich@in.tum.de
-		      April 26, 1999
-____________________________________________________________
-
+このファイルの日本語訳の部分、その他の変更部分は新規に作成したので上記２人に
+著作権があり、GPL に従って自由に配布できます。それ以外の変更のない部分につい
+ては原作の制限に従うものとします。
+このファイルに変更を加えた場合、このコメントに続き簡単な履歴を追加してください。
 */
 
+
 __TCLASS __BGN
   __NAME "GANYMEDE_CENTERED_SMALL_UP"
-  __NORMAL "ganymede/7"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/PAGER_RIGHT/border.cfg themes/Ganymede/borders/PAGER_RIGHT/border.cfg
--- themes/Ganymede.orig/borders/PAGER_RIGHT/border.cfg	1999-11-21 20:31:52.000000000 +0100
+++ themes/Ganymede/borders/PAGER_RIGHT/border.cfg	2000-12-28 19:09:45.000000000 +0100
@@ -1,15 +1,6 @@
-/*
-
-   T  H  E     G  A  N  Y  M  E  D  E      T  H  E  M  E
-
-	  (c) Chris Kreibich, kreibich@in.tum.de
-		      April 26, 1999
-____________________________________________________________
-
-*/
 __TCLASS __BGN
   __NAME "GANYMEDE_CENTERED_SMALL_DOWN"
-  __NORMAL "ganymede/7"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/RED/border.cfg themes/Ganymede/borders/RED/border.cfg
--- themes/Ganymede.orig/borders/RED/border.cfg	1999-10-17 03:43:02.000000000 +0200
+++ themes/Ganymede/borders/RED/border.cfg	2000-12-28 19:10:04.000000000 +0100
@@ -10,17 +10,17 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_RED"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 200 200
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 450
-  __NORMAL_ACTIVE "ganymede/10"
+  __NORMAL_ACTIVE "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 200
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 450
-  __NORMAL_ACTIVE_STICKY "ganymede/10"
+  __NORMAL_ACTIVE_STICKY "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 0
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/borders/ROTATED/border.cfg themes/Ganymede/borders/ROTATED/border.cfg
--- themes/Ganymede.orig/borders/ROTATED/border.cfg	1999-10-08 22:46:51.000000000 +0200
+++ themes/Ganymede/borders/ROTATED/border.cfg	2000-12-28 19:10:18.000000000 +0100
@@ -10,7 +10,7 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_DOWN"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/buttons/buttons.cfg themes/Ganymede/buttons/buttons.cfg
--- themes/Ganymede.orig/buttons/buttons.cfg	1999-11-21 20:31:16.000000000 +0100
+++ themes/Ganymede/buttons/buttons.cfg	2000-12-28 19:10:42.000000000 +0100
@@ -1,3 +1,4 @@
+
 /*
 
    T  H  E     G  A  N  Y  M  E  D  E      T  H  E  M  E
@@ -7,6 +8,7 @@
 ____________________________________________________________
 
 */
+
 #include <definitions>
 
 __E_CFG_VERSION 0
@@ -90,17 +92,17 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_CENTERED"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 240 240 240
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 512
-  __HILITED "ganymede/10"
+  __HILITED "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 512
-  __CLICKED "ganymede/10"
+  __CLICKED "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -109,19 +111,19 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_CENTERED_DOWN"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 240 240 240
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 512
   __ORIENTATION __FONT_TO_DOWN
-  __HILITED "ganymede/10"
+  __HILITED "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 512
   __ORIENTATION __FONT_TO_DOWN
-  __CLICKED "ganymede/10"
+  __CLICKED "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/coords/coords.cfg themes/Ganymede/coords/coords.cfg
--- themes/Ganymede.orig/coords/coords.cfg	1999-11-21 20:31:11.000000000 +0100
+++ themes/Ganymede/coords/coords.cfg	2000-12-28 19:10:54.000000000 +0100
@@ -1,15 +1,6 @@
-/*
-
-   T  H  E     G  A  N  Y  M  E  D  E      T  H  E  M  E
-
-	  (c) Chris Kreibich, kreibich@in.tum.de
-		      April 26, 1999
-____________________________________________________________
-
-*/
 __TCLASS __BGN
   __NAME "COORDS"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/dialogs/dialogs.cfg themes/Ganymede/dialogs/dialogs.cfg
--- themes/Ganymede.orig/dialogs/dialogs.cfg	1999-10-09 01:24:16.000000000 +0200
+++ themes/Ganymede/dialogs/dialogs.cfg	2000-12-28 19:11:40.000000000 +0100
@@ -7,13 +7,14 @@
 ____________________________________________________________
 
 */
+
 #include <definitions>
 
 __E_CFG_VERSION 0
 
 __TCLASS __BGN
   __NAME "DIALOG"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -22,7 +23,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_TEXT"
-  __NORMAL "ganymede_italic/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -31,7 +32,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_BUTTON"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -40,7 +41,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_RADIO_BUTTON"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -49,7 +50,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_TABLE"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -58,7 +59,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_CHECK_BUTTON"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -67,7 +68,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_AREA"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -76,7 +77,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_WIDGET_BUTTON"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -85,7 +86,7 @@
 
 __TCLASS __BGN
   __NAME "DIALOG_BUTTON"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/epplets/epplets.cfg themes/Ganymede/epplets/epplets.cfg
--- themes/Ganymede.orig/epplets/epplets.cfg	1999-12-13 12:44:20.000000000 +0100
+++ themes/Ganymede/epplets/epplets.cfg	2000-12-28 19:12:32.000000000 +0100
@@ -1,13 +1,3 @@
-/*
-
-   T  H  E     G  A  N  Y  M  E  D  E      T  H  E  M  E
-
-	  (c) Chris Kreibich, kreibich@in.tum.de
-		      April 26, 1999
-____________________________________________________________
-
-*/
-
 __ICLASS __BGN
   __NAME EPPLET_BACKGROUND_HORIZONTAL
   __NORMAL common/dock_u.png
@@ -35,18 +25,18 @@
   __NAME EPPLET_TOGGLEBUTTON_OFF
   __NORMAL epplets/images/button_u.png
   __EDGE_SCALING 1 1 1 1
-  __HILITED epplets/images/button_s.png
+  __HILITED epplets/images/button_c.png
   __EDGE_SCALING 1 1 1 1
-  __CLICKED epplets/images/button_c.png
+  __CLICKED epplets/images/button_s.png
   __EDGE_SCALING 1 1 1 1
   __PADDING 7 7 7 7
 __END
 
 __ICLASS __BGN
   __NAME EPPLET_TOGGLEBUTTON_ON
-  __NORMAL epplets/images/button_c.png
+  __NORMAL epplets/images/button_s.png
   __EDGE_SCALING 1 1 1 1
-  __HILITED epplets/images/button_s.png
+  __HILITED epplets/images/button_c.png
   __EDGE_SCALING 1 1 1 1
   __CLICKED epplets/images/button_u.png
   __EDGE_SCALING 1 1 1 1
@@ -138,7 +128,7 @@
 __END
 
 __ICLASS __BGN
-  __NAME EPPLET_ARROW_DOWN
+  __NAME EPPLET_ARROW_UP
   __NORMAL epplets/images/arrow_down_u.png
   __EDGE_SCALING 3 3 3 3
   __HILITED epplets/images/arrow_down_s.png
@@ -148,7 +138,7 @@
 __END
 
 __ICLASS __BGN
-  __NAME EPPLET_ARROW_UP
+  __NAME EPPLET_ARROW_DOWN
   __NORMAL epplets/images/arrow_up_u.png
   __EDGE_SCALING 3 3 3 3
   __HILITED epplets/images/arrow_up_s.png
@@ -309,7 +299,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_LABEL"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -318,7 +308,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_BUTTON"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -327,7 +317,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TOGGLEBUTTON_ON"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -336,7 +326,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TOGGLEBUTTON_OFF"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -345,7 +335,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_POPUP"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -354,7 +344,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TEXT_TINY"
-  __NORMAL "ganymede/6"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--10-*,-*-r-*--10-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -363,7 +353,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TEXT_MEDIUM"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
@@ -372,7 +362,7 @@
 
 __TCLASS __BGN
   __NAME "EPPLET_TEXT_LARGE"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--16-*,-*-r-*--16-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/iconbox/iconbox.cfg themes/Ganymede/iconbox/iconbox.cfg
--- themes/Ganymede.orig/iconbox/iconbox.cfg	1999-12-27 19:36:45.000000000 +0100
+++ themes/Ganymede/iconbox/iconbox.cfg	2000-12-28 19:13:02.000000000 +0100
@@ -10,7 +10,7 @@
 
 __TCLASS __BGN
   __NAME "GANYMEDE_TOOLTIP"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -66,49 +66,35 @@
 __END
 
 __ICLASS __BGN
-  __NAME ICONBOX_COVER_VERTICAL
-  __NORMAL "iconbox/images/cover.png"
-  __EDGE_SCALING   2 2 2 2
-  __PADDING        4 4 4 4
-__END
-
-__ICLASS __BGN
-  __NAME ICONBOX_COVER_HORIZONTAL
-  __NORMAL "iconbox/images/cover.png"
-  __EDGE_SCALING   2 2 2 2
-  __PADDING        4 4 4 4
-__END
-
-__ICLASS __BGN
   __NAME ICONBOX_SCROLLBAR_KNOB_HORIZONTAL
   __NORMAL "iconbox/images/scrollbar_horiz.png"
-  __EDGE_SCALING 10 10 1 1
+  __EDGE_SCALING 2 2 2 2
   __PADDING 2 2 2 2
 __END
 
 __ICLASS __BGN
   __NAME ICONBOX_SCROLLBAR_KNOB_VERTICAL
   __NORMAL "iconbox/images/scrollbar_vert.png"
-  __EDGE_SCALING 1 1 10 10
+  __EDGE_SCALING 2 2 2 2
   __PADDING 2 2 2 2
 __END
 
 __ICLASS __BGN
   __NAME ICONBOX_SCROLLKNOB_HORIZONTAL
-  __NORMAL "common/empty.png"
-  __EDGE_SCALING   1 1 1 1 
-  __CLICKED "common/empty.png"
-  __EDGE_SCALING   1 1 1 1
-  __PADDING        1 1 1 1
+  __NORMAL "dialogs/images/knob_normal.png"
+  __EDGE_SCALING   2 2 2 2
+  __CLICKED "dialogs/images/knob_clicked.png"
+  __EDGE_SCALING   2 2 2 2
+  __PADDING        2 2 2 2
 __END
 
 __ICLASS __BGN
   __NAME ICONBOX_SCROLLKNOB_VERTICAL
-  __NORMAL "common/empty.png"
-  __EDGE_SCALING   1 1 1 1
-  __CLICKED "common/empty.png"
-  __EDGE_SCALING   1 1 1 1 
-  __PADDING        1 1 1 1
+  __NORMAL "dialogs/images/knob_normal.png"
+  __EDGE_SCALING   2 2 2 2
+  __CLICKED "dialogs/images/knob_clicked.png"
+  __EDGE_SCALING   2 2 2 2
+  __PADDING        2 2 2 2
 __END
 
 __ICLASS __BGN
@@ -127,44 +113,36 @@
 
 __ICLASS __BGN
   __NAME ICONBOX_ARROW_UP
-  __NORMAL "iconbox/images/arrow_up1.png"
-  __EDGE_SCALING   2 2 2 2
-  __HILITED "iconbox/images/arrow_up2.png"
+  __NORMAL "iconbox/images/knob_normal.png"
   __EDGE_SCALING   2 2 2 2
-  __CLICKED "iconbox/images/arrow_up3.png"
+  __CLICKED "iconbox/images/knob_clicked.png"
   __EDGE_SCALING   2 2 2 2
   __PADDING        2 2 2 2
 __END
 
 __ICLASS __BGN
   __NAME ICONBOX_ARROW_DOWN
-  __NORMAL "iconbox/images/arrow_down1.png"
-  __EDGE_SCALING   2 2 2 2
-  __HILITED "iconbox/images/arrow_down2.png"
+  __NORMAL "iconbox/images/knob_normal.png"
   __EDGE_SCALING   2 2 2 2
-  __CLICKED "iconbox/images/arrow_down3.png"
+  __CLICKED "iconbox/images/knob_clicked.png"
   __EDGE_SCALING   2 2 2 2
   __PADDING        2 2 2 2
 __END
 
 __ICLASS __BGN
   __NAME ICONBOX_ARROW_LEFT
-  __NORMAL "iconbox/images/arrow_left1.png"
+  __NORMAL "iconbox/images/knob_normal.png"
   __EDGE_SCALING   2 2 2 2
-  __HILITED "iconbox/images/arrow_left2.png"
-  __EDGE_SCALING   2 2 2 2
-  __CLICKED "iconbox/images/arrow_left3.png"
+  __CLICKED "iconbox/images/knob_clicked.png"
   __EDGE_SCALING   2 2 2 2
   __PADDING        2 2 2 2
 __END
 
 __ICLASS __BGN
   __NAME ICONBOX_ARROW_RIGHT
-  __NORMAL "iconbox/images/arrow_right1.png"
-  __EDGE_SCALING   2 2 2 2
-  __HILITED "iconbox/images/arrow_right2.png"
+  __NORMAL "iconbox/images/knob_normal.png"
   __EDGE_SCALING   2 2 2 2
-  __CLICKED "iconbox/images/arrow_right3.png"
+  __CLICKED "iconbox/images/knob_clicked.png"
   __EDGE_SCALING   2 2 2 2
   __PADDING        2 2 2 2
 __END
@@ -228,7 +206,7 @@
 
 __TCLASS __BGN
   __NAME "ICONBOX_WINDOW_TITLE"
-  __NORMAL "ganymede/8"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
diff -Nur themes/Ganymede.orig/init/init.cfg themes/Ganymede/init/init.cfg
--- themes/Ganymede.orig/init/init.cfg	1999-09-27 19:39:52.000000000 +0200
+++ themes/Ganymede/init/init.cfg	2000-12-28 19:13:18.000000000 +0100
@@ -7,6 +7,7 @@
 ____________________________________________________________
 
 */
+
 #include <definitions>
 
 __E_CFG_VERSION 0
@@ -42,7 +43,7 @@
 
 __TCLASS __BGN
   __NAME "PROGRESS_TEXT"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -51,7 +52,7 @@
 
 __TCLASS __BGN
   __NAME "PROGRESS_TEXT_NUMBER"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 0 180 255
   __BACKGROUND_COLOR 0 50 128
diff -Nur themes/Ganymede.orig/menustyles/menustyles.cfg themes/Ganymede/menustyles/menustyles.cfg
--- themes/Ganymede.orig/menustyles/menustyles.cfg	1999-11-10 06:02:52.000000000 +0100
+++ themes/Ganymede/menustyles/menustyles.cfg	2000-12-28 19:13:45.000000000 +0100
@@ -7,18 +7,19 @@
 ____________________________________________________________
 
 */
+
 #include <definitions>
 
 __E_CFG_VERSION 0
 
 __TCLASS __BGN
   __NAME "GANYMEDE_MENU"
-  __NORMAL "ganymede/9"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 210 210 255
   __BACKGROUND_COLOR 0 0 0
   __JUSTIFICATION 512
-  __HILITED "ganymede/9"
+  __HILITED "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
@@ -101,7 +102,7 @@
 
 
 NORMAL_MENU_STYLE_VERTICAL("DEFAULT",    "MENU_BORDER",         "GANYMEDE_MENU", "MENU_FOCUS",   "MENUBACK",     "DEFAULT_MENU_SUB", 40)
-NORMAL_MENU_STYLE_VERTICAL("ROOT",       "MENU_BORDER",         "GANYMEDE_MENU", "MENU_FOCUS",   "MENUBACK",     "DEFAULT_MENU_SUB", 20)
+NORMAL_MENU_STYLE_VERTICAL("ROOT",       "MENU_BORDER",         "GANYMEDE_MENU", "MENU_FOCUS",   "MENUBACK",     "DEFAULT_MENU_SUB", 40)
 NORMAL_MENU_STYLE_VERTICAL("ROOT_IMAGE", "MENU_BORDER",         "GANYMEDE_MENU", "MENU_FOCUS",   "MENUBACK",     "DEFAULT_MENU_SUB", 10)
 NORMAL_MENU_STYLE_VERTICAL("EMPTY",      "MENU_NOTITLE", 	"GANYMEDE_MENU", "MENU_FOCUS",   "MENUBACK",     "DEFAULT_MENU_SUB", 25)
 NORMAL_MENU_STYLE_VERTICAL("CONFIG",     "MENU_NOTITLE", 	"GANYMEDE_MENU", "MENU_FOCUS",   "MENUBACK",     "DEFAULT_MENU_SUB", 4)
diff -Nur themes/Ganymede.orig/pager/pager.cfg themes/Ganymede/pager/pager.cfg
--- themes/Ganymede.orig/pager/pager.cfg	1999-11-21 20:30:47.000000000 +0100
+++ themes/Ganymede/pager/pager.cfg	2000-12-28 19:13:57.000000000 +0100
@@ -1,13 +1,3 @@
-/*
-
-   T  H  E     G  A  N  Y  M  E  D  E      T  H  E  M  E
-
-	  (c) Chris Kreibich, kreibich@in.tum.de
-		      April 26, 1999
-____________________________________________________________
-
-*/
-
 __ICLASS __BGN
   __NAME PAGER_SEL
   __NORMAL pager/images/pager_current.png
@@ -31,7 +21,7 @@
 
 __TCLASS __BGN
   __NAME "PAGER_WINDOW_TITLE"
-  __NORMAL "ganymede/10"
+  __NORMAL "-*-lucida-medium-r-normal-*-*-120-*-*-p-*-iso8859-1,-*-fixed-medium-r-normal--14-*,-*-r-*--14-*,*"
   __DRAWING_EFFECT __EFFECT_SHADOW
   __FORGROUND_COLOR 255 255 255
   __BACKGROUND_COLOR 0 0 0
